package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "lodash": "^4.17.4",
     "msgpack-lite": "^0.1.26",
+    "semver": "^5.6.0",
```

Comment:
I don't know why this is required. It comes with npm so the precommit check shouldn't complain. Maybe we can use node and the semver npm module to deal with semver versions in nvim's `autoload/`.

Reply:
I think it's good to have an explicit dependency for it just in case a user's environment isn't what you expect.